src/main/java/eu/europeana/metis/sandbox/consumer/workflow/TransformedConsumer.java

```diff
@@ -14,13 +14,18 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
+/**
+ * Consumes transformed events and performs internal validation to the contained record
+ * <br/>
+ * Publishes the result in the internally validated queue
+ */
 @Component
-public class TransformedConsumer {
+class TransformedConsumer {
```

Comment:
All those consumers are starting to look alike... is there perhaps something we can do to generalize?

Reply:
I did notice that about consumers, even attempt to do a generalization, but I failed with my first idea so I rolled back and need to think about it again, I will create a ticket to address this

Reply:
https://europeana.atlassian.net/browse/MET-2663